test/parallel/test-stream-pipeline.js

```diff
@@ -253,40 +253,6 @@ const net = require('net');
   });
 }
 
-{
-  const server = http.createServer((req, res) => {
-    pipeline(req, res, common.mustSucceed());
-  });
-
-  server.listen(0, () => {
-    const req = http.request({
-      port: server.address().port
-    });
-
-    let sent = 0;
-    const rs = new Readable({
-      read() {
-        if (sent++ > 10) {
-          return;
-        }
-        rs.push('hello');
-      }
-    });
-
-    pipeline(rs, req, common.mustCall(() => {
-      server.close();
-    }));
-
-    req.on('response', (res) => {
-      let cnt = 10;
-      res.on('data', () => {
-        cnt--;
-        if (cnt === 0) rs.destroy();
-      });
-    });
-  });
-}
```

Comment:
Haven't been able to make sense of this test. The way I see it, it should not have worked to begin with.

Reply:
This test fails with `ERR_PREMATURE_CLOSE` with the current changes, which makes sense to me. I am no sure either what was the behavior that had to be tested.